lua/harpoon/ui.lua

```diff
@@ -80,7 +81,7 @@ M.toggle_quick_menu = function()
     vim.api.nvim_buf_set_option(bufh, "filetype", "harpoon")
     vim.api.nvim_buf_set_option(bufh, "buftype", "acwrite")
     vim.api.nvim_buf_set_option(bufh, "bufhidden", "delete")
-    vim.cmd(string.format("autocmd BufWriteCmd <buffer=%s> :lua require('harpoon.ui').on_menu_save()", bufh))
+    --not sure if this line is still needed now?..
     vim.cmd(string.format("autocmd BufModifiedSet <buffer=%s> set nomodified", bufh))
```

Comment:
Both these au are used for `:wq` and `:w`.

Reply:
Then would it be logical to move these to lines to just before we call `vim.api.nvim_win_close(win_id, true) ` on line 56?
The problem seems to me that those two lines aren't called on closing.
```
vim.cmd(string.format("autocmd BufWriteCmd <buffer=%s> :lua require('harpoon.ui').on_menu_save()", bufh))
vim.cmd(string.format("autocmd BufModifiedSet <buffer=%s> set nomodified", bufh))
```